TestApp/ios/Podfile.lock

```diff
@@ -162,15 +162,15 @@ SPEC REPOS:
 
 EXTERNAL SOURCES:
   appcenter:
-    :path: "../node_modules/appcenter"
+    :path: "../node_modules/appcenter/ios"
```

Comment:
I am curious why the path changed given the fact https://github.com/microsoft/appcenter-sdk-react-native/pull/674 is not yet merged into develop. It shouldn't change, am I right?

Reply:
I think it is because we moved `.podspec` back to the `ios` folder in that [PR](https://github.com/microsoft/appcenter-sdk-react-native/pull/669/files)